include/swift/AST/DiagnosticsSema.def

```diff
@@ -1730,6 +1730,11 @@ WARNING(redundant_conformance_adhoc_conditional,none,
 NOTE(redundant_conformance_witness_ignored,none,
      "%0 %1 will not be used to satisfy the conformance to %2",
      (DescriptiveDeclKind, DeclName, DeclName))
+WARNING(redundant_conformance_warning,none,
+        "redundant conformance of %0 to %1", (Type, Type))
+NOTE(all_types_implicitly_conform_to,none,
+     "all types implicitly conform to %0", (Type))
+
```

Comment:
An alternative phrasing for the note would be "all types are subtypes of %0", which is more technically correct for `Any` as it's not actually a protocol you can conform to (just an empty protocol composition) – but I feel the current wording is more user friendly. Thoughts?